src/node/blockstorage.cpp

```diff
@@ -397,6 +404,16 @@ bool BlockManager::IsBlockPruned(const CBlockIndex* pblockindex)
     return (m_have_pruned && !(pblockindex->nStatus & BLOCK_HAVE_DATA) && pblockindex->nTx > 0);
 }
 
+const CBlockIndex* GetFirstStoredBlock(const CBlockIndex* start_block) {
```

Comment:
nit: clang-format

Reply:
Picked up these clang-formatting nits inside 8507260fd32914fa63 in #24150.